### PR TITLE
Add GetBucketRange() to return range of input value bucket

### PR DESCRIPTION
**Pull request by cheziyi**

### CODE REVIEW

File: PrismaDB-OrderPreservingBucketing/OpfInteger.cs [hunk 1]
```diff
@@ -46,6 +46,21 @@ public Int64 GetBucketId(Int64 value)
             return res;
         }
 
+        /// <summary>
+        /// Returns the bucket value range for min &lt= <c>value</c> &lt= max.
```
Comment (bazzilic) [inline PrismaDB-OrderPreservingBucketing/OpfInteger.cs]: I think this should have `;` - `&lt;`

Comment (cheziyi) [reply PrismaDB-OrderPreservingBucketing/OpfInteger.cs]: fixed

**Comment by bazzilic** [comment]
I’m not sure `uisubtract` is always safe. And I don’t quite understand why you are subtracting `bottom`.

**Comment by cheziyi** [comment]
Bottom is middle of uint64, negative numbers goes smaller, and positive goes larger. So subtracting bottom, if bucket number is smaller, will return back the negative bucket number, and for positive bring it down to 0.

**Comment by cheziyi** [comment]
`uisubtract` is not always safe, but for the purposes of BucketRange, it is. Want me to throw an error if values are too big?

**Comment by cheziyi** [comment]
`uisubtract()` functionality merged into `GetBucketRange()` to avoid confusion / improper usage.

**Review by bazzilic — APPROVED** [review]

**Pull request merged**